torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1331,6 +1331,14 @@ def create_unbacked_symint(self):
         self.var_to_range[symbol] = ValueRanges.unknown()
         return SymInt(SymNode(symbol, self, int, None))
 
+    def create_symfloat_arg(self, value, source):
+        assert isinstance(source, Source), f"{type(source)} {source}"
+        symbol = sympy.Symbol(f"d{len(self.var_to_val)}")
+        self.var_to_val[symbol] = sympy.Float(value)
+        self.var_to_sources[symbol] = [source]
+        self.var_to_range[symbol] = ValueRanges.unknown()
+        return SymFloat(SymNode(symbol, self, float, value))
```

Comment:
This is not currently used; was briefly used when I was playing around with SymFloat arguments to graphs.